src/runtime/pkg/katautils/config.go

```diff
@@ -1775,6 +1789,11 @@ func checkFactoryConfig(config oci.RuntimeConfig) error {
 // checkHypervisorConfig performs basic "sanity checks" on the hypervisor
 // config.
 func checkHypervisorConfig(config vc.HypervisorConfig) error {
+
+	if config.RemoteHypervisorSocket != "" {
```

Comment:
ok, if remote hypervisor then does not perform the remaining checks. But at this point the socket should be created or not? I'm figuring out whether to check also the socket or not.

Reply:
I'll be honest I'm not sure - I was assuming that in the config.go then it was just validating the config is set-up properly and not that it had been actioned i.e. the actual socket exists - assuming that is what you are getting at?

Reply:
`checkHypervisorConfig` does some checks like if the image is not empty, so I thought it wouldn't have sense to validate the remote socket existence. However, I am not sure if the socket file exist at this point.

Never mind, I think we can leave the code as is.